source/_docs/autostart/init.d.markdown

```diff
@@ -36,6 +36,9 @@ This logrotate script at `/etc/logrotate.d/homeassistant` will create an outage
 
 ```
 /var/log/homeassistant/home-assistant.log
```

Comment:
Is this formatting valid for specifying the location of the file which should contain the next block of text?

Reply:
Adjusted, since it is indeed not part of that file.